eng/pipelines/sdk-update.yml

```diff
@@ -199,8 +199,15 @@ stages:
           Matrix: $[dependencies.Generate_Job_Matrix.outputs['generate_job_matrix.matrix']]
           AutorestCSharpVersion: $(AutorestCSharpVersion)
           BranchName: $(BranchName)
+          ContinueOnError: true
```

Comment:
Is it possible that we could make the step to fail when there is errors, but we still could run the following step `Create_PR`?

The reason I ask this is that in practice, we might get some random glitch of failure, and if the step fails, we could only rerun the step and work this around, such as failure like this:
![image](https://github.com/Azure/autorest.csharp/assets/10554446/e61e8d27-b923-4a75-904f-2a3c7a4e7716)
And to work issues like this around, rerunning the whole thing might not solve it (every PR has the possibility to have this issue when a bunch of jobs running together), and when every step is green, there is no button to rerun the failed steps:
![image](https://github.com/Azure/autorest.csharp/assets/10554446/24262dc5-ed55-49a0-8a52-488105af255a)
And when the steps fails, we could:
![image](https://github.com/Azure/autorest.csharp/assets/10554446/698bc754-bce7-4b0c-9e04-2c755c6d6e0c)